jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerClient.java

```diff
@@ -42,6 +47,33 @@
 /** Calls out to the {@code docker} CLI. */
 public class DockerClient {
 
+  /**
+   * Contains the size, image ID, and diff IDs of an image inspected with {@code docker inspect}.
+   */
+  public static class InspectResults {
```

Comment:
Can we use a more specific name, like `CachedImageDetails`?  This is capturing the results of our `inspect -f ...`, not the general `docker inspect` case.

Reply:
I'll name it `DockerImageDetails` so we don't confuse it with Jib's cache.